save-backend/src/main/kotlin/org/cqfn/save/backend/configs/H2.kt

```diff
@@ -0,0 +1,35 @@
+package org.cqfn.save.backend.configs
+
+import org.slf4j.LoggerFactory
+import org.h2.tools.Server
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.event.ContextClosedEvent
+import org.springframework.context.event.ContextRefreshedEvent
+import org.springframework.context.event.EventListener
+import org.springframework.stereotype.Component
+import java.sql.SQLException
+
+/**
+ * This class is used for creation of h2 console on port 8081
+ */
+@Component
+class H2 {
+    private var webServer: Server? = null
```

Comment:
Is explicit configuration of console still required with `spring.h2.console.enabled=true`? Won't it be automagically created at `/h2-console`, as stated [here](https://www.baeldung.com/spring-boot-h2-database#h2-console)?

Reply:
Since we are using webflux we need it

Reply:
Oh, ok.